app/settings/surveys/targeted-surveys/targeted-survey-edit.html

```diff
@@ -180,9 +188,28 @@ <h2 class="stepper-heading">
                                 <span class="stepper-badge">4</span>
                                 <span translate="survey.targeted_survey.publish"></span>
                             </h2>
+                            <div class="alert" ng-show="isActiveStep(4)">
+                                <p translate="survey.targeted_survey.publish_description" translate-values="{questions: targetedSurvey.stepThree.questions.length, numbers:finalNumbers.goodNumbers.length, sms: sms, cost:cost}"></p>
+                            </div>
+                            <div class="form-field switch checked">
```

Comment:
These should be hidden until you're in step 4

Reply:
fixed!